### ISSUE TITLE
Virtualization

### ISSUE THREAD

**Issue by jaksco** [gregnb/mui-datatables#428]
I noticed that you are looking on adding virtualization. I'm also interested in adding this to my project.

`react-window` might be another good option for a virtualization library. It was made by the developer of react-virtualized but it has a much simpler and refined API

https://react-window.now.sh/#/examples/grid/variable-size

**Comment by gregnb** [comment]
Absolutely! Would love to work in virtualization for tables that have pagination={false}. If you would like to take up a POC and see how it turns out I would love to take a look

**Comment by L3mmAa** [comment]
Can you guys guess when you are able to implement it?

**Comment by msteitle** [comment]
Bump. Material-UI Tables has an example for adding virtualization in its docs.
https://material-ui.com/components/tables/#virtualized-table

**Comment by ariel-upstream** [comment]
Do we have an example for mui-datatables?

**Comment by UseMuse** [comment]
how to implement virtualization? infinite-scrolling - increases the number of objects, but it is necessary to constantly update the number of displayed objects

**Comment by ariel-upstream** [comment]
Is the feature of virtualization planned to be implemented in the near future?

**Comment by genefever** [comment]
Hi any updates on this?

**Comment by MGMehdi** [comment]
Bump

**Comment by MGMehdi** [comment]
I finally gave up and made my own board with MaterialUI and [react-virtuoso](https://virtuoso.dev/). 
[Virtuoso](https://virtuoso.dev/mui-table-virtual-scroll/) even has an example to implement MUI arrays and list and is very easy to set up compared to react-window. It also supports horizontal scroll and can be responsive

**Comment by MGMehdi** [comment]
Here is an exemple with Virtuso, MUI and table filter
https://codesandbox.io/s/virtuoso-mui-filter-f5xtfr
